src/hotspot/share/cds/cdsConfig.hpp

```diff
@@ -61,6 +69,7 @@ class CDSConfig : public AllStatic {
   static bool      is_dumping_dynamic_archive()              { return CDS_ONLY(_is_dumping_dynamic_archive) NOT_CDS(false); }
   static void  enable_dumping_dynamic_archive()              { CDS_ONLY(_is_dumping_dynamic_archive = true); }
   static void disable_dumping_dynamic_archive()              { CDS_ONLY(_is_dumping_dynamic_archive = false); }
+  static bool      is_using_archive()                        NOT_CDS_RETURN_(false);
```

Comment:
Could you fix the alignment of the method names here?

Reply:
Since several people are confused by the alignment style (align same words to the right), I fixed the grouping of the functions so that the text is aligned to the left, as in most header files.

Please take a look at [ae0e0ac](https://github.com/openjdk/jdk/pull/18095/commits/ae0e0acc72bcf1bba81dc4218a64766eb2f2549a) -- it's best viewed on GutHub with white spaces hidden.